scripts/monitor_submission.py

```diff
@@ -46,7 +46,17 @@ def monitor_submission(terra_workspace, terra_project, submission_id, sleep_time
         # save metadata
         save_name = 'monitor_submission_metadata.json'
         with open(save_name, 'w') as f:
-            f.write(json.dumps(submission_metadata))
+            # terra_details to be appended to json
+            terra_details = {"terra_workspace": terra_workspace, "terra_project": terra_project}
+
+            # loading JSON submission_metadata:
+            submission_data = json.loads(submission_metadata)
+
+            # appending the terra_details
+            submission_data.update(terra_details)
```

Comment:
`.update` doesn't overwrite the existing json, just appends the contents of `terra_details`?

Reply:
Yup

Reply:
could you make that clear in a comment just so our future selves aren't confused! otherwise this is good to go!